### PR TITLE
Merge imputed filled posts 

### PR DESCRIPTION
**Pull request by MarionJHolloway**
# Description
Add a function to merge the columns from interpolation and extrapolation in the estimate missing ascwds data job

# How to test
Unit tests passing
Run in branch: https://eu-west-2.console.aws.amazon.com/states/home?region=eu-west-2#/v2/executions/details/arn:aws:states:eu-west-2:344210435447:execution:merge-imputed-filled-posts-Ind-CQC-Filled-Post-Estimates-Pipeline:cf98c00d-a70b-4e51-8668-10c2c16a8c31

# Developer checklist
- [x] Unit test
- [x] Linked to Trello ticket: https://trello.com/c/MbRox2mW/726-create-ascwdsfilledpostsimputed-column-for-ml-model-to-use-must
- [x] Documentation up to date

### CODE REVIEW

File: tests/unit/test_estimate_missing_ascwds_ind_cqc_filled_posts.py [hunk 1]
```diff
@@ -65,5 +65,30 @@ def test_number_of_days_in_rolling_average_value(self):
...
+        test_df = self.spark.createDataFrame(
+            Data.merge_imputed_columns_rows, Schemas.merge_imputed_columns_schema
+        )
+        returned_df = job.merge_imputed_columns(test_df)
+        expected_df = self.spark.createDataFrame(
+            Data.expected_merge_imputed_columns_rows,
+            Schemas.expected_merge_imputed_columns_schema,
+        )
+        returned_df.show()
+        expected_df.show()
```
Comment (garypolzin) [inline tests/unit/test_estimate_missing_ascwds_ind_cqc_filled_posts.py:82]: remove :)

File: tests/unit/test_estimate_missing_ascwds_ind_cqc_filled_posts.py [hunk 2]
```diff
@@ -65,5 +65,30 @@ def test_number_of_days_in_rolling_average_value(self):
...
+        returned_data = returned_df.sort(IndCQC.location_id).collect()
+        expected_data = expected_df.collect()
```
Comment (garypolzin) [inline tests/unit/test_estimate_missing_ascwds_ind_cqc_filled_posts.py:84]: probably safer to sort this too

**Review by garypolzin — APPROVED** [review]
2 minor comments so I'll approve now :)

File: tests/unit/test_estimate_missing_ascwds_ind_cqc_filled_posts.py (hunk 2 above)
Comment (MarionJHolloway) [reply tests/unit/test_estimate_missing_ascwds_ind_cqc_filled_posts.py:84]: I can do, but sorts are expensive and nothing is done to the expected data that might un-sort it. It's just created and collected?

**Pull request merged**